Code/Python/Sudoku_Solver.py

```diff
@@ -0,0 +1,88 @@
+# Title: Sudoku Solver
+# Link: https://leetcode.com/problems/sudoku-solver/
```

Comment:
we need an appropriate description of the issue.
Links are not supposed to be added.